### PR TITLE
Guess the number

### PR DESCRIPTION
**Pull request by Moniarchy**

### CODE REVIEW

File: spec/guess-the-number-spec.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+  it( 'exists', () => {
+    expect( guessTheNumber ).to.be.a( 'function' )
+  })
+
+  context( 'randomNumberGenerator()', () =>
+    it( 'generates a random number n', () => {
+      expect( randomNumberGenerator() ).to.be.a( 'number' )
+    }),
+    it( 'generates a number whose value is between 0 and 100', () => {
```
Comment (jrobcodes) [inline spec/guess-the-number-spec.js]: Testing randomness is hard - in this test, it's not guaranteed that the number will *always* be in the range between 0 and 100...

File: src/guess-the-number.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+const readlineSync = require('readline-sync')
+const prompt = readlineSync.question
+
+const guessTheNumber = guess => {
+  const randomNumber = randomNumberGenerator()
+    if ( guess > randomNumber ) {
```
Comment (jrobcodes) [inline src/guess-the-number.js]: The indentation here makes this code look like it is part of another block within `guessTheNumber` - I actually read it as a function that returned a function until second glance...  Consider indenting appropriately, and adding whitespace between the `randomNumber` declaration, and the body of this function (the `if` statement).

File: src/guess-the-number.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+      console.log ('Your guess was too high!')
+    } else if ( guess < randomNumber ) {
+      console.log ('Your guess was too low!')
+    } else {
+      console.log ('You got it!')
+    }
+    console.log( 'The computer says: ' + randomNumber )
+}
+
+const randomNumberGenerator = () => {
```
Comment (jrobcodes) [inline src/guess-the-number.js]: You may omit the braces and `return` keyword if you'd like (I prefer it; it feels a little more concise, and less visually noise to me).

File: spec/guess-the-number-spec.js (hunk 1 above)
Comment (Moniarchy) [reply spec/guess-the-number-spec.js]: I couldn't think of a way to check for randomness, so I opted to make sure that the number will always be in the appropriate range. 

http://chaijs.com/api/bdd/#method_within

Am I misunderstanding the proper use of the within assertion? It "asserts that the target is within a range."

File: src/guess-the-number.js (hunk 2 above)
Comment (Moniarchy) [reply src/guess-the-number.js]: Will do--this was leftover indentation from when these conditionals were part of another function block.

**Pull request merged**